app/views/pair_requests/_pair_request.html.erb

```diff
@@ -5,19 +5,17 @@
   </p>
 
   <p>
-    <strong>Acceptor:</strong>
-    <% if pair_request.acceptor.present? %>
-      <% if pair_request.acceptor == current_user %>
-        <%= link_to "Unaccept", pair_request_acceptance_path(@pair_request), data: { turbo_method: :delete }, class: "btn btn-danger" %>
-      <% else %>
-        <%= pair_request.acceptor.email %>
-      <% end %>
-    <% else %>
-      <% if pair_request.author == current_user %>
-        Not yet accepted
-      <% else %>
-        <%= link_to "Accept this request", pair_request_acceptances_path(pair_request), data: { turbo_method: :post }, class: "btn btn-success" %>
-      <% end %>
+    <strong>Invitee: <%= pair_request.invitee.email %></strong>
+  </p>
+
+  <p>
+    <strong>Status: <%= pair_request.status %></strong>
+    <% if  AcceptancePolicy.new(current_user, pair_request).create? %>
```

Comment:
This is kind of the only way we could find to trigger a policy in the views. The `policy` helper uses metaprogramming to look up the object's policy by name and doesn't offer a way to overwrite this.

Here's an issue referencing this behavior: https://github.com/varvet/pundit/issues/630

Reply:
lol.... yeah I disagree with what pundit did. We can see how things are going and potentially define a refinement or something to fix this ourselves.

For now what you have is perfect.